app/plugin_api_test.go

```diff
@@ -173,6 +174,33 @@ func TestPluginAPIUpdateUserStatus(t *testing.T) {
 	assert.Nil(t, status)
 }
 
+func TestPluginAPIGetFile(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+
+	// check existing user first
+	teamId := model.NewId()
+	channelId := model.NewId()
+	userId := model.NewId()
+	filename := "testGetFile"
+	fileData := []byte("Hello World")
+	info, err := th.App.DoUploadFile(time.Date(2007, 2, 4, 1, 2, 3, 4, time.Local), teamId, channelId, userId, filename, fileData)
```

Comment:
you can use `th.BasicTeam`/`.BasicChannel`/`BasicUser`

Reply:
you can do this:
```
uploadTime := time.Date(2007, 2, 4, 1, 2, 3, 4, time.Local)
filename := "testGetFile"
fileData := []byte("Hello World")
info, err := th.App.DoUploadFile(uploadTime, th.BasicTeam.Id, th.BasicChannel.Id, th.BasicUser.Id, filename, fileData)
```

Reply:
th.BasicTeam is a struct that holds lot of info, like id, Name, ...

Reply:
Fixed this 👍